README.md

```diff
@@ -19,3 +19,23 @@ The Tungsten Hackathon channeled the energy and excitement of DFINITY's internal
 [Shield](https://github.com/crusso/shield) - An app connecting at-risk people with nearby volunteers.
 
 [Hero](https://github.com/stanleygjones/hackathon) - A random "killer app" generator inspired by the Wu-Tang Name Generator.
+
+
+## Motoko Libraries
```

Comment:
Can we add Vessel and the VSCode plugin too?

Reply:
This isn't open either, AFAIK -- https://github.com/dfinity/dfinity-lab/vscode-motoko

Reply:
cc @kritzcreek

Reply:
I focused on getting `base` out there. I'll take a look at the VSCode plugin as my next OS project.

Reply:
Okay, I've done a pass of revisions to update the list to only have stuff that is public now.

We can add things as they each become public, hopefully soon.